docker-compose.yml

```diff
@@ -4,9 +4,10 @@ services:
   wildfly:
     build: .
     container_name: "wildfly"
-    network_mode: "host"
-    hostname: "localhost"
     command: "/opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0"
+    ports:
+      - "8080:8080"
```

Comment:
Out of curiosity, does it check if port 8080 is available and then switch to 9990 if it's occupied? 🤔

Reply:
If you want to access the management interface (wildfly's) then you need port 9990. To configure wildfly itself.